File: lesson3/python3/homework/homework3_10c.py
#!/usr/bin/python3
#: lesson3/python3/homework/homework3_10c.py

x = float(input("Введите первую координату => "))
y = float(input("Введите вторую координату => "))

if x <= 0 and y <= 0 and y <= -x:
    print("попали.")
else:
    print("непопали.")

File: lesson3/c_c++/homework/homework3_10c.cpp
//: lesson3/c_c++/homework/homework3_10c.cpp
#include <iostream>

int main() {
    float x, y;
    std::cout << "Введите первую координату => ";
    std::cin >> x;
    std::cout << "Введите вторую координату => ";
    std::cin >> y;

    if (x <= 0 && y <= 0 && y <= -x) {
        std::cout << "попали.\n";
    } else {
        std::cout << "непопали.\n";
    }
    return 0;
}
